feat: Release notes for Codacy Cloud December 2023 DOCS-448 (#1995)
codacy/docs: docs/release-notes/cloud/cloud-2023-12.md

---
rss_title: Codacy release notes RSS feed
rss_href: /feed_rss_created.xml
description: Release notes for Codacy Cloud December 2023.
included_jira_versions: ['2023.12']
codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/7.10.123
codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/7.10.158
---

# Cloud December 2023

These release notes are for the Codacy Cloud updates during December 2023.

📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!

## Product enhancements

-   The [Security and risk management feature](../../organizations/managing-security-and-risk.md), previously only available to organization admins and security managers, is now open to all organization members and shows information related to the repositories that each member follows. (TAROT-2442)

> -   The [Codacy Coverage Reporter](https://coverage.codacy.com/) now only allows reports with filenames matching the files on your Git provider. (ALA-716)

Reviewer: * I think we can link to our release note here.
* I updated this to sound less "threatening" 😄 

```suggestion
-   When calculating coverage data, the [new coverage engine](./cloud-2023-11-23-new-coverage-engine-status-checks.md) now ignores files in the coverage report that are not in the associated Git repository. (ALA-716)
```

> -   When calculating coverage data, the [new coverage engine](./cloud-2023-11-23-new-coverage-engine-status-checks.md) now ignores files in the coverage report that are not in the associated Git repository. (ALA-716)

Author: ```suggestion
-   When calculating coverage data, the [new coverage engine](./cloud-2023-11-23-new-coverage-engine-status-checks.md) now ignores files in the coverage report that aren't in the associated Git repository. (ALA-716)
```
